src/sentry.ts

```diff
@@ -199,6 +202,15 @@ export default class SentryService {
   /*
   * Helper Functions
   */
+  async setTags(tags: { [key: string]: number | string }): Promise<void> {
```

Comment:
remove since we set tags in start transaction

Reply:
Removed `setTags` function